.local.mockery.yaml

```diff
@@ -1,5 +1,4 @@
 all: True
-case: underscore
```

Comment:
Apparently, this config field is unsupported with the latest version. After re-generate mocks, it seems the the underscore case is used by default.

Errors:
![image](https://github.com/DataDog/chaos-controller/assets/8859832/f3aced67-b063-4691-b828-dec51177cd52)